ferrocene/doc/sphinx-shared-resources/src/ferrocene_standalone_make_cli.py

```diff
@@ -32,6 +33,9 @@ def build_docs(root, builder, clear, serve, debug):
         # Enable parallel builds:
         args += ["-j", "auto"]
     if clear:
+        html_path = Path(f"{dest}/html")
```

Comment:
I have a few comments on this:

* The path `"{dest}/html"` is not always correct, if the builder being passed to the function is not "html".
* We already define the output directory [here](https://github.com/ferrocene/ferrocene/pull/1313/files#diff-88bde951e18af58a80087724e0456631fa2b27158e084dcb6592d44387245298R59), to ensure this doesn't break when moving the output directory somewhere else we should use the same variable.
* Python's `pathlib` has a nice syntax to join paths: the division operator! You can do `dest / "html"` as an equivalent of the code you wrote, or Rust's `dest.join("html")`.

Reply:
Thank you!
I packed all in an `output_dir` so we don't forget to modify both.